test/harry/main/org/apache/cassandra/harry/model/SelectHelper.java

```diff
@@ -305,7 +305,8 @@ public static List<ResultSetRow> execute(SystemUnderTest sut, OpSelectors.Clock
 
     public static List<ResultSetRow> execute(SystemUnderTest sut, OpSelectors.Clock clock, Query query, Set<ColumnSpec<?>> columns)
     {
-        CompiledStatement compiled = query.toSelectStatement(columns, true);
+        boolean isAccord = query.schemaSpec.isWriteTimeFromAccord();
+        CompiledStatement compiled = isAccord ? query.toWildcardSelectStatement() : query.toSelectStatement(columns, true);
```

Comment:
we did this in one other location; the local checker... but I switched off local checker